plejd/PlejdBLEHandler.js

```diff
@@ -1,3 +1,4 @@
+// @ts-ignore
```

Comment:
`ts-ignore` is for typescript, should not be used here (also at least one more place)

Reply:
Yea, there were curly red markings in the following line, therefore I accepted the proposal to ignore the following line. Should have left it alone.

I do not have access to a developer environment until later this week, so either I fix it later or you just skip those changes at merge time.